service-base/src/main/java/org/eclipse/hono/service/resourcelimits/PrometheusBasedResourceLimitChecks.java

```diff
@@ -219,11 +220,11 @@ private Span createSpan(final String name, final SpanContext parent, final Tenan
 
         if (tenant.getResourceLimits() == null) {
             traceItems.put(Fields.EVENT, "no resource limits configured");
-            LOG.trace("no resource limits configured for tenant [{}]", tenant.getTenantId());
+            LOG.warn("no resource limits configured for tenant [{}]", tenant.getTenantId());
```

Comment:
this is an expected situation which we explicitly handle. If at all, the level might be increased to DEBUG but I do not see any reason why we should flood the log with this information.

Reply:
Sorry, just noticed this as well. Shouldn't have gone into the commit.